docs/articles/nunit/getting-started/installation.md

```diff
@@ -1,43 +1,149 @@
 We recommend you only use this approach when running under the control of a script on your build server.
 
-## NUnit3TestAdapter
+See here for more details of using the [NUnit Console app](https://github.com/nunit/nunit-console#readme)
 
-If you want to run NUnit tests automated on a clean machine without any installations (e.g. TFS build agent) - and you're using Visual Studio 2012 or later, use this package.
+## NUnit3TestAdapter
 
-It's based on [the Visual Studio Test Adapter](xref:vstestadapterinstallation) and provides a compiled NUnit3 Visual Studio Test Adapter.
+If you want to run NUnit tests automated on a clean machine without any installations (e.g. Azure Devops or Github actions build agent) - and you're using Visual Studio 2012 or later, use this package.
 
-### Using NuGet NUnitLite Package
+See [the Visual Studio Test Adapter](xref:vstestadapterinstallation) for details.
 
-The NUnitLite approach provides a way to run NUnit tests without a full install of the NUnit runner and test engine assemblies. Only the framework and a small runner program are installed. Note that this is currently the only way to run tests under SilverLight 5.0 or the compact framework.
+Note that if you have used any of the top 4 approaches, you already have this package installed, and it will work on any CI build system.
 
-In Visual Studio, from the Tools menu, select NuGet Package Manager | Manage NuGet packages for solution...
+### Using NuGet NUnitLite Package
 
-Scroll or use search to locate the **NUnitLite Version 3** and install it. The package should be referenced by each of your test assemblies, but not by any others.
+The NUnitLite approach provides a way to run NUnit tests without a full install of the NUnit runner and test engine assemblies. Only the framework and a small runner program are installed. If you want to experiment with NUnit, debug something in NUnit, this can be a good way to go.
 
-As a result of your installation, a file named "Program.cs" will have been copied into each of your test projects. NUnitLite test projects are console applications and this file contains the Main() program for your tests. If you already have a Main() in some other class, you may delete the file. This file also contains comments describing how the main should call the NUnitLite runner.
+Follow the instructions for [NUnitLite](xref:nunitlite) to install the package and create a test assembly. You will need to add a reference to the NUnitLite package in your test assembly.
 
 To run your tests, simply run your executable test assembly. No other runner is needed.
 
-### Downloading the Zip File
+Note that since this approach includes a Program.cs it can not be combined with the ordinary Unit test project, as that would give dual entry points, making the compiler confused.
```

Comment:
```suggestion
Note that since this approach includes a `Program.cs` it can not be combined with the ordinary unit test project, as that would create two entry points, which would confuse the compiler.
```